ci_env.yml

```diff
@@ -1,7 +1,7 @@
 name: human-dynamics-estimation
 channels:
-  - conda-forge
   - robotology
+  - conda-forge
```

Comment:
conda-forge should be higher in the channel list

Reply:
Thanks, I was trying to switch the order to understand if it was causing the issue, I'll roll back to previous order.

Reply:
Order reverted in https://github.com/robotology/human-dynamics-estimation/pull/371/commits/cfd3acf45f8a14a41984f485c075db0b3ff21ebf